webpack.config.js

```diff
@@ -3,6 +3,13 @@ let productionBuild = (process.env.production_build == 1);
 let version = "v0.0.1";
```

Comment:
I find that `DefinePlugin` is still somehow needed. When we bundle all the reactjs together, it seems that react won't know what is `process`, because `process` is a global variable in Node environment. Since react runs in browser, `process` will not exist there. Therefore, when I already renamed then env variable to `NODE_ENV`, there will be error saying that `process is not defined` when I run `node ./index.js`.

I googled a bit on this, and turned out that many people actually had this issues, and `DefinePlugin` seems to be a workaround for this. Btw, `DefinePlugin` is shipped with `webpack` itself, so this won't introduce extra dependencies. 

Now I've made it look like this 
```javascript
    new webpack.DefinePlugin({
        'process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV),
        'global': {}
    })
```